src/math/math.cr

```diff
@@ -618,6 +620,12 @@ module Math
   end
 
   # :ditto:
+  def ldexp(value : Float32 | Float64, exp : Int)
```

Comment:
Shouldn't `BigFloat` be allowed too?
ditto elsewhere

Reply:
#11007